src/content/docs/dropins/cart/tutorials/add-product-lines-to-cart-summary.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Add custom product lines to the cart summary
+description: Learn how to add custom lines related to products in the cart summary.
+---
+
+import Diagram from '@components/Diagram.astro';
+import Callouts from '@components/Callouts.astro';
+import Aside from '@components/Aside.astro';
+import { Tabs, TabItem, Code, Steps } from '@astrojs/starlight/components';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+
+This tutorial describes how to make the following customizations to the `CartSummaryList` container using the Adobe Commerce Boilerplate:
+
+- Add text from a custom product attribute
+- Display promotional information in the footer of each product in the cart
+
+## Prerequisites
+
+This tutorial requires that you create the following entities in the Adobe Commerce Admin:
+
+- A custom product attribute. Here, the product attribute is assigned the label `Shipping Notes`, and the **Catalog Input Type for Store Owner* is set to **Text Field**. You can optionally set the **Used for Sorting in Product Listing** option to **Yes** to make the product attribute easier to find applicable products in the Products grid. [Product attributes overview](https://experienceleague.adobe.com/en/docs/commerce-admin/catalog/product-attributes/product-attributes) describes how to create a custom product attribute.
+
+  In addition, you must assign the product attribute to one or more products. In this tutorial, the text fields will contain the strings "These item(s) are available to ship on Nov 1, 2024" and "FINAL SALE: This item ships separately and is ineligible for return.".
+
+- A custom cart price rule. In this tutorial, a cart price rule named `25% Off $75+ with Code BOO24` has been created. Its definition defines the coupon code, the discount amount, and the conditions that must be met to apply the discount. [Cart price rules overview](https://experienceleague.adobe.com/en/docs/commerce-admin/marketing/promotions/cart-rules/price-rules-cart) describes how to create a cart price rule.
+
+<Tasks>
+
+<Task>
+
+## Add text from a custom product attribute
```

Comment:
The `Task` component is designed to [recognize H3 only](https://github.com/commerce-docs/microsite-commerce-storefront/blob/develop/src/components/Task.astro#L36-L63). I'm sure we can change it, but we may not want to since the pattern we've been using in other tutorials is "Step-by-step" as an H2 and then the following `<Task>` sections as H3 so that the blue "Step n" badge appears at the top left of the task box.

Consider normalizing with other tutorials.